docs/translations.md

```diff
@@ -1,43 +1,672 @@
+languages.
+
+[fluent]: https://projectfluent.org/
+[ftl-bundles]: https://github.com/mozilla/fx-private-relay/blob/main/privaterelay/ftl_bundles.py
+[ga-l10n-sync]: https://github.com/mozilla/fx-private-relay/actions/workflows/l10n-sync.yml
+[mozilla-l10n/fx-private-relay-add-on-l10n]: https://github.com/mozilla-l10n/fx-private-relay-add-on-l10n
+[mozilla-l10n/sumo-l10n]: https://github.com/mozilla-l10n/sumo-l10n
+[mozilla-l10n/www-l10n]: https://github.com/mozilla-l10n/www-l10n
+[mozilla-l10n]: https://github.com/mozilla-l10n
+[mozilla/addons-frontend]: https://github.com/mozilla/addons-frontend/locale
+[mozilla/fxa-content-server-l10n]: https://github.com/mozilla/fxa-content-server-l10n
+[privaterelay/locales]: https://github.com/mozilla/fx-private-relay/tree/main/privaterelay
+[relay-l10n-en]: https://github.com/mozilla-l10n/fx-private-relay-l10n/tree/main/en
+
+## The Technical Details of Region Selection
+
+Relay premium plans are not available world-wide, only in some regions. As of September
+2023, the premium email service is available in 34 regions, and the phone service in 2
+regions. This section details the technologies used to distinguish by region.
+
+### Identifying the User's Region
+
+In the stage and production deployments, the load balancer sets a `X-Client-Region`
+header that identifies the user's region, based on their IP address. There is no
+way for a user to override this region selection. A tester can use a [VPN][moz-vpn] to
+change their region.
+
+When the `X-Client-Region` header is not available, then the code falls back to the
+`Accept-Language` header to guess the user's region. The `X-Client-Region` header is not
+available in the development deployment or in local development, and may not be
+available in stage and production deployments for some IP addresses.
+
+The `Accept-Language` fallback looks for a language code with a regional variant, such
+as `es-MX` for Mexican Spanish, and uses that region. If the language code does not
+include a region, then the code guesses a probable region from a [lookup table][].
+The [Language-Territory Information][cldr-lti] from the Unicode
+[Common Locale Data Repository][cldr] (CLDR) [Supplemental Data][cldr-sup], which
+estimates of language speakers by region, provides the data for the lookup table.
+
+[cldr-lti]: https://www.unicode.org/cldr/charts/42/supplemental/language_territory_information.html
+[cldr-sup]: https://www.unicode.org/cldr/charts/42/supplemental/index.html
+[cldr]: https://cldr.unicode.org/
+[lookup table]: https://github.com/mozilla/fx-private-relay/blob/f5c5ebf568639810db45de1ebb69f2498600d58c/privaterelay/utils.py#L72-L216
+[moz-vpn]: https://www.mozilla.org/products/vpn
+
+### Identifying Available Plans and Prices
+
+The user's region determines what premium plans are available. The API sends this
+data from [/api/v1/runtime_data][], and the website adjusts content based on
```

Comment:
comment (non-blocking): FWIW, we say "front end" 4 times, "front-end" 1 time, and "website" 16 times. Note: "front end" and "front-end" always refer to Relay front end, while "website" refers to:
* Relay website 5 times (including this one)
* FXA website: 1 time
* Firefox Subscriptions website: 1 time
* other websites: 5 times

Reply:
The new text:

> The user’s region determines what premium plans are available. The API sends this data in [/api/v1/runtime_data](https://relay.firefox.com/api/v1/runtime_data). The front end adjusts content based on availability in the user’s region.

I use "website" in the section "How a User Experiences Relay", since "website" is user language. I use website to talk about other people's websites, including FxA and subscriptions.

Later, I use "front end" when talking about the JS code that runs on the Relay website, "back end" for Python on the server. I ended up not using "front-end" as an adjective.

I was convinced by this discussion:

https://english.stackexchange.com/questions/34447/is-it-front-end-frontend-or-front-end.
 
We could decide on something else